README.es.md

```diff
@@ -0,0 +1,99 @@
+# sway [![](https://api.travis-ci.org/SirCmpwn/sway.svg)](https://travis-ci.org/SirCmpwn/sway) [![Dona con fosspay](https://drewdevault.com/donate/static/donate-with-fosspay.png)](https://drewdevault.com/donate?project=4)
+
+[**English**](https://github.com/SirCmpwn/sway/blob/master/README.md#sway--) - [日本語](https://github.com/SirCmpwn/sway/blob/master/README.ja.md#sway--) - [Deutsch](https://github.com/SirCmpwn/sway/blob/master/README.de.md#sway--)
+
+"**S**irCmpwn's **Way**land compositor" es un compositor de [Wayland](http://wayland.freedesktop.org/) 
+compatible con i3 **en desarrollo**.
+Lee el [FAQ](https://github.com/SirCmpwn/sway/wiki). Únete al
+[canal IRC](http://webchat.freenode.net/?channels=sway&uio=d4) (#sway en
+irc.freenode.net).
+
+[![](https://sr.ht/ICd5.png)](https://sr.ht/ICd5.png)
+
+Si quieres soportar el desarrollo de Sway, puedes contribuir a [mi página de
+Patreon](https://patreon.com/sircmpwn) o puedes contribuir [recompensas](https://github.com/SirCmpwn/sway/issues/986)
```

Comment:
Can you translate this to something to the effect of "on Patreon" rather that first person?